setup.py

```diff
@@ -163,7 +163,11 @@ def _redirect_stderr(to_fd, original_stderr_fd):
 
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 if on_rtd:
+    logger.warning('Building on Read the Docs with Cython enabled.')
     args.use_cython = True
+    for cython_cpp_file in glob.glob('freud/*.cpp'):
+        logger.warning('Deleting {}'.format(cython_cpp_file))
```

Comment:
We can make this silent if preferred, but I think having it loud is better because it will only ever show up on RTD and helps us track the code's behavior.

Reply:
I'm fine keeping it loud.